### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by ashishbhate**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+
+## License
+
+MIT © [AgFlow](https://www.agflow.com)
+
+<a title="AgFlow" href="https://www.agflow.com">
+  <img border="0" alt="AgFlow" src="https://raw.githubusercontent.com/agflow/logos/master/agflow-logo.png" width="130" height="40" />
+</a>
+
+[![Build Status](https://travis-ci.org/agflow/docker-misspell.svg?branch=master)](https://travis-ci.org/agflow/docker-misspell)
```
Comment (ferhatelmas) [inline README.md]: can we put it next to headline ?

Comment (ashishbhate) [reply README.md]: Do you mean next to the "docker-misspell" heading?

Comment (ferhatelmas) [reply README.md]: yes

Comment (ashishbhate) [reply README.md]: done

**Pull request merged**